packages/blade/src/components/Badge/__tests__/Badge.web.test.tsx

```diff
@@ -59,7 +59,7 @@ describe('<Badge />', () => {
 
   it('should render Badge with bold font', () => {
     const label = 'LABEL';
-    const { container } = renderWithTheme(<Badge fontWeight="bold">{label}</Badge>);
+    const { container } = renderWithTheme(<Badge fontWeight="semibold">{label}</Badge>);
```

Comment:
no snapshots changed? 🤔

Reply:
yes. They are in this branch - https://github.com/razorpay/blade/tree/rebranding/typography-snapshots

I'll merge these changes first to `rebranding/master` and then update snapshots again